src/sharedHooks/useSuggestions/useOnlineSuggestions.ts

```diff
@@ -115,9 +134,9 @@ const useOnlineSuggestions = (
     safeRealmWrite( realm, ( ) => {
       mappedTaxa.forEach( remoteTaxon => {
         realm.create(
-          "Taxon",
+          Taxon,
```

Comment:
This is a new pattern that hasn't been used in the codebase before. I would like to be included in the learning process as to why this is also possible to use (or if it is better than the existing pattern).

Reply:
mmmm, yeah, I didn't realize that was totally new at that time. I've pulled this back for now and added as a team topic. I'm pro, it's just not something we've done before.